corona-warn-app/cwa-app-android
# TAN Hotline no. not clickable (EXPOSUREAPP-2551)

## Description
Made the actual TAN hotline number clickable. When clicked, the hotline number will be dialled in the same fashion as when the button is clicked. 

- Declared textview clickable within xml layout
- Reused dial logic from button and used for now clickable text  

### Test
- Navigate to hotline test submission option
- Select the blue hotline number
- Hotline number should be dialed. 

Closes JIRA ticket EXPOSUREAPP-2551
